scripts/postinstall.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Automatically copy over required settings for vscode
+if [ ! -f .vscode/settings.json ];
+	then cp .vscode/settings.json.required .vscode/settings.json ;
```

Comment:
Might be useful to add a little message here that shows up in the users console? Maybe saying something like "Automatically created .vscode/settings.json - this overwrites some users settings, please check & modify as required" or similar?